skydance/protocol.py

```diff
@@ -20,6 +20,7 @@
 # See: https://github.com/tomasbedrich/home-assistant-skydance/issues/1
 _COMMAND_MAGIC = bytes.fromhex("80 00 80 e1 80 00 00")
 
+DEVICE_BASE_TYPE_NORMAL = 0x80
```

Comment:
Can you please add comment what this means?

Reply:
This is from decompiled sources:

```java
package com.lxit.skydance.bean;

public class DeviceBaseType {
    public static final byte DEVICE_BASE_TYPE_DMX = 4;
    public static final byte DEVICE_BASE_TYPE_DMX_WIFI = 0x44;
    public static final String DEVICE_BASE_TYPE_DMX_WIFI_CONTROLLER_NAME = "WiFi-DMX";
    public static final byte DEVICE_BASE_TYPE_NORMAL = 0x80 // <--- here is this constant definition 
    public static final byte DEVICE_BASE_TYPE_PWD = 1;
    public static final byte DEVICE_BASE_TYPE_PWD_WIFI = 65;
    public static final String DEVICE_BASE_TYPE_PWD_WIFI_CONTROLLER_NAME = "WiFi-CV4";
    public static final byte DEVICE_BASE_TYPE_SPI = 2;
    public static final byte DEVICE_BASE_TYPE_SPI_WIFI = 66;
    public static final String DEVICE_BASE_TYPE_SPI_WIFI_CONTROLLER_NAME = "WiFi-SPI";
    public static final byte DEVICE_BASE_TYPE_WIFI_RELAY = -64;
    public static final String DEVICE_BASE_TYPE_WIFI_RELAY_CONTROLLER_NAME = "WiFi-Relay";
}
```